src/compass/utils/h5_helpers.py

```diff
@@ -243,7 +254,7 @@ def add_dataset_and_attrs(group, name, value, attr_dict):
     add_dataset_and_attrs(id_proc_group, 'product_level', '2', {})
     add_dataset_and_attrs(id_proc_group, 'product_type', 'CSLC_S1', {})
     add_dataset_and_attrs(id_proc_group, 'processing_datetime',
-                          datetime.now().strftime(time_str_fmt),
+                          datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
I chose not to use `TIME_STR_FMT = '%Y-%m-%d %H:%M:%S.%f'` because I don't think this piece of information requires detail of fractional seconds. I can change if you and others disagree.

Since the format string ` '%Y-%m-%d %H:%M:%S` is single use, I saw no need to declare it elsewhere.

Reply:
Thanks for the clarifying about your intention. Then it seems like most of my comments about the formatting can be solved by fixing `TIME_STR_FMT = '%Y-%m-%d %H:%M:%S'` into `TIME_STR_FMT = '%Y-%m-%d %H:%M:%S.%f'`